receiver/hostmetricsreceiver/internal/scraper/memoryscraper/metadata.yaml

```
@@ -8,7 +8,7 @@ attributes:
   state:
     description: Breakdown of memory usage by type.
     type: string
-    enum: [buffered, cached, inactive, free, slab_reclaimable, slab_unreclaimable, used]
+    enum: [total, buffered, cached, inactive, free, slab_reclaimable, slab_unreclaimable, used]
```

```suggestion
    enum: [buffered, cached, free, inactive, slab_reclaimable, slab_unreclaimable, used, total]
```
This was _almost_ in alphabetical order, we may as well sort all of it.